src/services/users/index.js

```diff
@@ -152,11 +154,44 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (token) => {
+	try {
+		const decoded = jwt.verify(token, config('TOKEN_SECRET'))
+		const {email} = decoded
+		const result = await  db.query(
+			`SELECT * FROM user_invites
+		WHERE email = $1 
+		`,
+			[email]
+		)
+		const user = result.rows[0]
+		if (!user) {
+			throw customError(RefreshTokenIsInvalidError)
```

Comment:
not sure how there being no user is related to `RefreshToken`. lets use the invalid invite error then

Reply:
you are right